src/components/Restoration/BusinessName.vue

```diff
@@ -1,132 +1,70 @@
 <template>
   <div id="business-name">
-    <div class="section-container" :class="{'invalid-section': invalidSection}">
-      <!-- Business Name -->
+    <!-- *** TODO: remove before flight -->
+    <!-- <pre>companyName={{ companyName }}</pre> -->
+    <!-- <pre>getNameRequestApprovedName={{ getNameRequestApprovedName }}</pre> -->
+    <!-- <pre>isCorrectingName={{ isCorrectingName }}</pre> -->
+
+    <!-- Editing Mode -->
```

Comment:
Using new shared CorrectName component for editing.

Below, using existing NameRequestInfo component for displaying.